scripts/makefile_helper

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+
+# SPDX-License-Identifier: GPL-2.0
+# Copyright (C) 2022-present Team LibreELEC (https://libreelec.tv)
+
+# default variables
+CLEAN="false"
+DISTCLEAN="false"
+ROOT="${PWD}"
+
+
+# Just interested in how the builder set BUILD_DIR...
+if [ -f "${ROOT}/.libreelec/options" ]; then
+  . "${ROOT}/.libreelec/options"
+fi
+
+if [ -f "${HOME}/.libreelec/options" ]; then
+  . "${HOME}/.libreelec/options"
+fi
+
+BUILD_DIR="${BUILD_DIR:-${ROOT}}"
+
+
+# CLI argument handling
+case $1 in
+  -c|--clean)
+  CLEAN="true"
+  ;;
+  -d|--distclean)
+  DISTCLEAN="true"
+  ;;
+  *)
+  echo "error: ${0}: unsupported option on CLI; aborting"
+  exit 1
+  ;;
+esac
+
+
+# cleanup tasks
+if [ ${CLEAN} = "true" ]; then
```

Comment:
Little pointless to have extra variable set to execute just one command outside of case statement?

Reply:
Force of habit for longer scripts to separate CLI handling from performing work. Reworked, thanks.